server/sonar-server/src/test/java/org/sonar/server/component/ComponentTesting.java

```diff
@@ -107,7 +109,32 @@ public static ComponentDto newDeveloper(String name) {
       .setEnabled(true);
   }
 
-  private static ComponentDto newComponent(String uuid, ComponentDto module) {
+  public static ComponentDto newView(String uuid) {
+    return newProjectDto(uuid)
+      .setUuid(uuid)
+      .setScope(Scopes.PROJECT)
+      .setQualifier(Qualifiers.VIEW);
+  }
+
+  public static ComponentDto newTechnicalProject(ComponentDto project, ComponentDto view) {
```

Comment:
This DTO is about COPY_RESOURCE_ID column, so method name should be related: newProjectCopy(project, parent)